src/editors/data/services/cms/api.ts

```diff
@@ -30,7 +43,19 @@ export const apiMethods = {
   fetchStudioView: ({ blockId, studioEndpointUrl }) => get(
     urls.blockStudioView({ studioEndpointUrl, blockId }),
   ),
-  fetchImages: ({ learningContextId, studioEndpointUrl, pageNumber }) => {
+  fetchImages: ({
+    learningContextId,
+    studioEndpointUrl,
+    pageNumber,
+  }): Promise<{ data: AssetResponse & Pagination }> => {
+    if (learningContextId.startsWith('lib:')) {
```

Comment:
suggestion: There's a few places where you make this particular check, and so I'm wondering if this is a good time to add some opaque key string utils?

Reply:
> I'm wondering if this is a good time to add some opaque key string utils?

Good idea. I added some to `src/generic/key-utils`, and we can always add more in the future.